Sources/Controllers/2.0/OATripRecordingSettingsViewController.mm

```diff
@@ -129,13 +129,10 @@ -(UIView *) getTopView
     return _navBarView;
 }
 
--(UIView *) getMiddleView
-{
-    return _tableView;
-}
-
 - (void) setupView
 {
+    if ([self.backButton isDirectionRTL])
+        self.backButton.transform = CGAffineTransformMakeRotation(M_PI);
```

Comment:
Why transform?

Reply:
because "self.backButton.imageView.image = self.backButton.imageView.image.imageFlippedForRightToLeftLayoutDirection" is rewriting image, and when we go to inner settings and go back, it flips image 1 more time.